app/src/main/java/org/holtnet/officialimperialbusiness/ExamActivity.java

```diff
@@ -80,11 +89,19 @@ public void onClick(View v) {
         });
     }
 
+    /**
+     * More documentation! :)  Probably best to leave the onClick as part of the XML and not a style
+     * So that the editor doesn't think that the method is not being used (when it is)
+     *
+     */
+
     public void onQuestionsRadioButtonClicked(View view) {
         //Verify if the correct radio button answer is checked. If it is, set the corresponding variable to true.
         // Otherwise set it to false.
 
-        boolean checked = ((RadioButton) view).isChecked();
+
+        // Your buttons could have been named better than they are.  Your code is like a puzzle.
+        boolean checked = ((RadioButton) view).isChecked();  // Why have this?
```

Comment:
It was in the example in the developer documentation and I believe it simplifies my code, i.e. makes my code repeat less often / use fewer characters.